eslint.config.mjs

```diff
@@ -0,0 +1,35 @@
+import bpmnIoPlugin from 'eslint-plugin-bpmn-io';
+
+const files = {
+  expose: [
+    '**/*.js',
+    '*.js',
+  ],
+  build: [
+    '*.mjs',
+  ]
+};
+
+/** @type {import('eslint').Linter.Config[]} */
+export default [
+  ...bpmnIoPlugin.configs.browser.map(config => {
+    return {
+      ...config,
+      ignores: files.build,
+      files: files.expose,
+      languageOptions: {
+        sourceType: 'commonjs'
+      },
+      environment: {
+        browser: true,
```

Comment:
This is setup by the [browser config, no](https://github.com/bpmn-io/eslint-plugin-bpmn-io/blob/main/configs/browser.js)?